pkg/upstream/cluster/loadbalancer.go

```diff
@@ -129,5 +132,127 @@ func (lb *roundRobinLoadBalancer) HostNum(metadata api.MetadataMatchCriteria) in
 	return len(lb.hosts.Hosts())
 }
 
+const default_choice = 2
+
+// leastActiveRequestLoadBalancer choose the host with the least active request
+type leastActiveRequestLoadBalancer struct {
+	*EdfLoadBalancer
+	choice uint32
+}
+
+func newleastActiveRequestLoadBalancer(info types.ClusterInfo, hosts types.HostSet) types.LoadBalancer {
+	lb := &leastActiveRequestLoadBalancer{}
+	if info != nil && info.LbConfig() != nil {
+		lb.choice = info.LbConfig().(*v2.LeastRequestLbConfig).ChoiceCount
+	} else {
+		lb.choice = default_choice
+	}
+	lb.EdfLoadBalancer = newEdfLoadBalancerLoadBalancer(hosts, lb.unweightChooseHost)
+	return lb
+}
+
+func (lb *leastActiveRequestLoadBalancer) unweightChooseHost(context types.LoadBalancerContext) types.Host {
+
+	healthyHosts := lb.hosts.HealthyHosts()
```

Comment:
Now `lb.hosts.HealthyHosts()` has been deprecated in this [PR](https://github.com/mosn/mosn/commit/d903bd10ececee4706605f01a73ec193ac9191bd#diff-8d8fb39636bed5042ff869294470b69c), So we should update the logical and you can refer [it](https://github.com/mosn/mosn/blob/master/pkg/upstream/cluster/loadbalancer.go#L72).

Reply:
done~